src/components/MainPage.js

```diff
@@ -0,0 +1,140 @@
+/* @flow */
+import React, { Component, Fragment } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import 'App.css';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as MoviesActions from '../actions/actions';

```

Comment:
The absolute path here.

Also, about importing of a `*` - usually it is bad for three shaking. So use it with wisdom :)